test/strategies.js

```diff
@@ -2365,6 +2365,147 @@ const createAaveFLV3RepayStrategy = () => {
     return aaveV3RepayStrategy.encodeForDsProxyCall();
 };
 
+const createAaveV3RepayOnPriceStrategy = () => {
+    const aaveV3RepayOnPriceStrategy = new dfs.Strategy('AaveV3RepayOnPrice');
+
+    aaveV3RepayOnPriceStrategy.addSubSlot('&collAsset', 'address');
+    aaveV3RepayOnPriceStrategy.addSubSlot('&collAssetId', 'uint16');
+    aaveV3RepayOnPriceStrategy.addSubSlot('&debtAsset', 'address');
+    aaveV3RepayOnPriceStrategy.addSubSlot('&debtAssetId', 'uint16');
+    aaveV3RepayOnPriceStrategy.addSubSlot('&marketAddr', 'address');
+    aaveV3RepayOnPriceStrategy.addSubSlot('&targetRatio', 'uint256');
+    aaveV3RepayOnPriceStrategy.addSubSlot('&useOnBehalf', 'bool');
+
+    const aaveV3Trigger = new dfs.triggers.AaveV3RatioTrigger('0', '0', '0');
+    aaveV3RepayOnPriceStrategy.addTrigger(aaveV3Trigger);
+
+    const withdrawAction = new dfs.actions.aaveV3.AaveV3WithdrawAction(
+        '&useDefaultMarket', // set to true hardcoded
+        '%market', // hardcoded because default market is true
+        '%amount', // must stay variable
```

Comment:
We want to use specific market user is subbed to, and leave 'useDefaultMarket' to false value:
```diff
- '&useDefaultMarket', // set to true hardcoded
- '%market', // hardcoded because default market is true

+ '%useDefaultMarket', // hardcode to false
+ '&marketAddr',

```